consensus/e2e_tests/state_sync_test.go

```diff
@@ -274,31 +261,20 @@ func TestStateSync_UnsyncedPeerSyncs_Success(t *testing.T) {
 		require.Equal(t, typesCons.NodeId(0), nodeState.LeaderId)
 	}
 
-	unsyncedNode.GetBus().GetConsensusModule().SetAggregatedStateSyncMetadata(uint64(1), testHeight, string(leaderPK.Address()))
-
+	unsyncedNode.GetBus().GetConsensusModule().PushStateSyncMetadataResponse(uint64(1), testHeight)
 	for _, message := range newRoundMessages {
 		P2PBroadcast(t, pocketNodes, message)
 	}
 	advanceTime(t, clockMock, 10*time.Millisecond)
 
-	// 2. Propose
-	numExpectedMsgs := numberOfValidators
+	numExpectedMsgs := numValidators
 	_, err = WaitForNetworkConsensusEvents(t, clockMock, eventsChannel, consensus.Prepare, consensus.Propose, numExpectedMsgs, 500, true)
 	require.NoError(t, err)
 
-	advanceTime(t, clockMock, 10*time.Millisecond)
-	for nodeId, pocketNode := range pocketNodes {
-		nodeState := GetConsensusNodeState(pocketNode)
-		fmt.Printf("Node state, node id: %d, height: %d, round: %d, step: %d, leaderId: %d \n", nodeId, nodeState.Height, nodeState.Round, nodeState.Step, nodeState.LeaderId)
-		assertNodeConsensusView(t, nodeId,
-			typesCons.ConsensusNodeState{
-				Height: testHeight,
-				Step:   uint8(consensus.Prepare),
-				Round:  uint8(currentRound),
-			},
-			nodeState)
-		require.Equal(t, leaderId, nodeState.LeaderId)
-	}
+	// TODO (#352) This function will be updated once state sync implementation is complete
+	waitForNodeToSync(t, clockMock, eventsChannel, unsyncedNode, pocketNodes, testHeight)
+
+	// TODO (#352) Add height check once state sync implmentation is complete
```

Comment:
Let's follow the same pattern everywhere

```suggestion
	// TODO(#352): Add height check once state sync implmentation is complete
```